Gordon360/Models/CCT/AccountPhotoURL.cs

```diff
@@ -0,0 +1,24 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT
+{
+    [Keyless]
+    public partial class AccountPhotoURL
+    {
+        [StringLength(10)]
+        [Unicode(false)]
+        public string Gordonid { get; set; }
+        [StringLength(86)]
+        [Unicode(false)]
+        public string PhotoURL { get; set; }
+        [StringLength(86)]
+        [Unicode(false)]
+        public string PrefPhotoURL { get; set; }
+    }
+}
```

Comment:
Probably avoid adding DB models for objects that you are not using in this PR. There is a chance that they haven't been created in production by the time that you merge it in and that could break the site.

This one is actually my view that I created in train and I will be pushing it later, but it is best practice for you to not include it in your branch.

Reply:
You shouldn't uncheck all of the DBO models. You should have the specific ones checked that we need in 360. You can expand the tree further. My guess is that one of you went in and checked all of them at some point accidentally, now you have no way to track what is strictly necessary.

Reply:
Actually @bennettforkner I believe that EF Core is currently configured to bring in all tables and view from CCT, they did not make this change in their branch.

I also don't think it really matters if they bring in unrelated DB models, all it does is add a tiny bit of dead code to the build, which is a non-issue as far as I'm concerned.